src/panels/developer-tools/state/developer-tools-state.ts

```diff
@@ -0,0 +1,682 @@
+import { addHours } from "date-fns/esm";
+import "@material/mwc-button";
+import {
+  mdiClipboardTextMultipleOutline,
+  mdiInformationOutline,
+  mdiRefresh,
+} from "@mdi/js";
+import { CSSResultGroup, LitElement, css, html, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { formatDateTimeWithSeconds } from "../../../common/datetime/format_date_time";
+import { computeRTL } from "../../../common/util/compute_rtl";
+import { escapeRegExp } from "../../../common/string/escape_regexp";
+import { copyToClipboard } from "../../../common/util/copy-clipboard";
+import "../../../components/entity/ha-entity-picker";
+import "../../../components/ha-yaml-editor";
+import "../../../components/ha-icon-button";
+import "../../../components/ha-svg-icon";
+import "../../../components/ha-checkbox";
+import "../../../components/ha-tip";
+import "../../../components/ha-alert";
+import "../../../components/search-input";
+import "../../../components/ha-expansion-panel";
+import { showAlertDialog } from "../../../dialogs/generic/show-dialog-box";
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { toggleAttribute } from "../../../common/dom/toggle_attribute";
+
+@customElement("developer-tools-state")
+class HaPanelDevState extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() private _error: string = "";
```

Comment:
Instead of assigning empty strings to all these, probably better to leave them undefined to expose any potential issues

```suggestion
  @state() private _error?: string;
```

Reply:
I've also done tried this in the developer events page, but the text field just picks up "undefined" and puts that in the field. Therefore setting it to empty.